openbas-model/src/main/java/io/openbas/database/model/ChallengeAttemptId.java

```diff
@@ -0,0 +1,21 @@
+package io.openbas.database.model;
+
+import jakarta.persistence.Embeddable;
+import java.io.Serial;
+import java.io.Serializable;
+import lombok.Data;
+
+@Data
+@Embeddable
+public class ChallengeAttemptId implements Serializable {
+
+  @Serial private static final long serialVersionUID = 1L;
+
+  private String challengeId;
+  private String injectStatusId;
+  private String userId;
+
+  public ChallengeAttemptId() {
```

Comment:
Default constructor is not needed if there are no others, I believe?

Reply:
You are right !